pkgs/development/libraries/libusb1/default.nix

```diff
@@ -33,7 +33,12 @@ stdenv.mkDerivation rec {
 
   NIX_LDFLAGS = stdenv.lib.optionalString stdenv.isLinux "-lgcc_s";
 
-  preFixup = stdenv.lib.optionalString stdenv.isLinux ''
+  configureFlags =
+    # We use `isLinux` here only to avoid mass rebuilds for Darwin, where
+    # disabling udev happens automatically. Remove `isLinux` at next big change!
+    stdenv.lib.optional (stdenv.isLinux && !enableSystemd) "--disable-udev";
```

Comment:
Should probably not re-add the `stdenv.isLinux` here, per the comment.

Reply:
I wasn't sure what next big change is or whether now is a big change. Should I remove it?

Reply:
The mass rebuild of udev and everything depending on it here are caused by the configuration options changing. Since this logic was already removed in b3862e2bcf5b658730c206ba12339acbe47b7945, the mass Darwin rebuild will have already happened then, and reintroducing the unnecessary option would instead result in another mass rebuild.

Reply:
In other words, the big change already accidentally happened :)

Reply:
That makes sense. I changed it.

Reply:
Or actually, in b3862e2 all of the logic was removed, so Darwin wouldn't use the `--disable-udev` rule, as it did before b3862e2 and 370d483.